src/bots/ama/models/Event.js

```diff
@@ -1,9 +1,12 @@
 import mongoose from 'mongoose';
 
-const EventSchema = mongoose.Schema({
-    title: String,
-    date: Date
-}, {collections: 'Event', autoCreate: true});
+const EventSchema = mongoose.Schema(
```

Comment:
This seems inconsistent.

Reply:
What is inconsistent?

Reply:
The schema appearance, but upon second glance I guess is fine